sos/policies/runtimes/lxd.py

```diff
@@ -94,7 +94,8 @@ def get_volumes(self):
             if out['status'] == 0:
                 out_json = json.loads(out['output'])
                 for profile in out_json:
-                    if profile['name'] == 'default':
+                    if (profile['name'] == 'default' and
+                            'root' in profile['devices']):
```

Comment:
This had to be added as part of the fix of bringing the build of the deb package build on a minimal OS installation. But this also fixes an issue if the LXD was never initialised and lxd package is actually installed.